metadata_backend/api/auth.py

```diff
@@ -1,7 +1,7 @@
 """Handle Access for request and OIDC workflow."""
 
 import time
-from typing import Dict, List, Union
+from typing import Any, Dict, List, Union
```

Comment:
I know we don't like `Any`, but it comes from `oidcrp` which is oblivious to the userinfo response type `userinfo: Dict[str, Any]`, so I put this in there :smile: